src/vendor/algolia/algolia.js

```diff
@@ -112,14 +115,16 @@ const buildOfferTypesPredicate = offerTypes => {
   }
 }
 
-const buildGeolocationParameter = (aroundRadius, geolocation) => {
+const buildGeolocationParameter = (aroundRadius, geolocation, isSearchAroundMe) => {
   if (geolocation) {
     const { longitude, latitude } = geolocation
     if (latitude && longitude) {
       const aroundRadiusInMeters = aroundRadius * 1000
       return {
         aroundLatLng: `${latitude}, ${longitude}`,
-        aroundRadius: aroundRadius ? aroundRadiusInMeters : FILTERS.UNLIMITED_RADIUS,
+        aroundRadius: isSearchAroundMe ?
+          aroundRadiusInMeters > 0 ? aroundRadiusInMeters : FILTERS.UNLIMITED_RADIUS
+          : FILTERS.UNLIMITED_RADIUS
```

Comment:
```suggestion
        aroundRadius: isSearchAroundMe &&
          aroundRadiusInMeters > 0 ? aroundRadiusInMeters : FILTERS.UNLIMITED_RADIUS
```
C'est pas équivalent ça ?